app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -93,16 +93,16 @@ fun Menu() {
             horizontalArrangement = Arrangement.Center,
             verticalAlignment = Alignment.CenterVertically
         ) {
-            Text(text= countryName, fontSize = 18.sp, modifier = Modifier.padding(end = 8.dp))
+            Text(text = menuName, fontSize = 18.sp, modifier = Modifier.padding(end = 8.dp))
             Icon(imageVector = Icons.Filled.ArrowDropDown, contentDescription = "")
-            DropdownMenu(expanded = expanded, onDismissRequest = {expanded = false}) {
```

Comment:
Proper formatting can be attained using the hotkey ctr+alt+l on windows. Formatting best practices can be found at https://kotlinlang.org/docs/coding-conventions.html#formatting